src/Uuid/Uuid.php

```diff
@@ -214,6 +189,43 @@ final public static function for(
 		};
 	}
 
+	/**
+	 * Creates a UUID object from a UUID string
+	 * @since 6.0.0
+	 */
+	final public static function from(
+		string $uuid,
+		string|array|null $scheme = null,
```

Comment:
`Suggestion`: `Uuid::is()` uses the name `$type`, I'd use the same here for consistency:

```suggestion
		string|array|null $type = null,
```